app/src/main/java/co/nimblehq/chorn/survey/ui/theme/AppTypography.kt

```diff
@@ -2,9 +2,24 @@ package co.nimblehq.chorn.survey.ui.theme
 
 import androidx.compose.material.Typography
 import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.*
+import androidx.compose.ui.unit.sp
+import co.nimblehq.chorn.survey.R
+
+internal val NeuzeitFontFamily = FontFamily(
+    Font(R.font.neuzeit_regular, FontWeight.Normal),
+    Font(R.font.neuzeit_bold, FontWeight.Bold)
+)
 
 private val Typography = Typography(
-    // Custom typography here
+    button = TextStyle(
+        fontSize = 17.sp,
+        fontFamily = NeuzeitFontFamily,
+        fontWeight = FontWeight.Bold,
+        color = ChineseBlack,
+        letterSpacing = 0.sp
+    )
```

Comment:
Can we bring this to `buttonTextStyle` in `AppStyle`?

```suggestion
    button = buttonTextStyle
```

// In `AppStyle

```kotlin
@Composable
fun buttonTextStyle = TextStyle(
        fontSize = 17.sp,
        fontFamily = NeuzeitFontFamily,
        fontWeight = FontWeight.Bold,
        color = ChineseBlack,
        letterSpacing = 0.sp
    )
```

Reply:
@chornerman But actually, we will assign the font for the Typography like this:

```suggestion
val Typography = Typography(
    defaultFontFamily = NeuzeitFontFamily
)
```
So I think we can add one more `defaultFontFamily` to set the default font. And once we define the text style, we can omit the `fontFamily` like this:

```kotlin
@Composable
fun buttonTextStyle = TextStyle(
        fontSize = 17.sp,
        fontWeight = FontWeight.Bold,
        color = ChineseBlack,
        letterSpacing = 0.sp
    )
```